api/src/main/java/edu/cornell/mannlib/vitro/webapp/application/VitroHomeDirectory.java

```diff
@@ -210,4 +217,79 @@ public String toString() {
 		}
 	}
 
+    /**
+     * Populates home directory with home files, excluding the rdf directory
+     */
+    public void populate() {
+        File vhdDir = getPath().toFile();
+
+        if (!vhdDir.isDirectory() || vhdDir.list() == null) {
+            throw new RuntimeException("Application home dir is not a directory! " + vhdDir);
+        }
+        
+        if (!vhdDir.canWrite()) {
+            throw new RuntimeException("Application home dir is not writable! " + vhdDir);
+        }
+        try {
+            copy(vhdDir);
+        } catch(Exception e) {
+            log.error(e, e);
+            throw new RuntimeException("Failed to copy home files! " + vhdDir);
+        }
+        log.info("Copied home files to " + vhdDir.toPath());
+        
+    }
+
+    /**
+     * Create home directory
+     */
+    private static void createHomeDirectory(Path vhdDir) {
+        try {
+            vhdDir.toFile().mkdirs();
+        } catch (Exception e) {
+            log.error(e, e);
+            throw new RuntimeException("Failed to create home directory " + vhdDir);
+        }
+    }
+
+    /**
+     * Copy file from home source to home destination
+     */
+    private void copy(File homeDestination) throws IOException {
+        File homeSrcPath = new File(getHomeSrcPath(ctx));
+        File[] contents = homeSrcPath.listFiles();
+        for (File child : contents) {
+            if (!isExcluded(child)) {
+                FileUtils.copyDirectory(child, homeDestination);
+            }
+        }
+    }
+
+    /**
+     * Test if file name is excluded from copying
+     * 
+     * @return true if file should be excluded
+     */
+    private boolean isExcluded(File child) {
+        if (excludedHomeFiles.contains(child.getName())) {
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Get source home file directory path
+     * 
+     * @return source home files directory path
+     */
+    public static String getHomeSrcPath(ServletContext context) {
+        String location = "/WEB-INF/resources/home-files";
+        String realPath = context.getRealPath(location);
+        if (realPath == null) {
+            log.error("Application home files not found in: " + location);
+            throw new RuntimeException("Application home files not found in: " + location);
```

Comment:
The throwable could be added to the RuntimeException instantiation and the error log can be removed.